proto/ocpaths.proto

```diff
@@ -26,6 +26,19 @@ package openconfig.ocpaths;
 
 option go_package = "github.com/openconfig/featureprofiles/proto/ocpaths_go_proto;ocpaths";
 
+// Release is at minimum a timestamp for associated with an artifact.
+// By convention, itt is expected to uniquely identify and artifact
+message Release {
+  // the date/time of the relase
+  google.protobuf.Timestamp date = 1;
+
+  // the optional version of the release, typically numbers and dots.
+  optional string version = 2;
+
+  // the optional moniker (nickname) for the release
+  optional string name = 3;
+}
```

Comment:
https://github.com/openconfig/featureprofiles/blob/13c822568b5abf9a3efbf5dd78f43a3e826feb5f/proto/nosimage.proto#L39-L43

there are no nicknames for releases in nosimage.proto

Reply:
Good point. The concept of the release triplet was not spread around to the rest of the
schema. You could rightly make the point that it should be. I have some reviewers coming
from G that I'll ask about that. Thanks for raising.

Reply:
per comments, this does have a constraint on the value to match /system/state/software-version

Reply:
I believe the version of the NOS should remain in the NOSImageProfile and is not needed here.

Reply:
We need to be clear if we are talking about the OC release (https://github.com/openconfig/public/releases)  or the NOS release (vendor operating system version)?   

Because this release is in the OCPath.proto, it should in my mind be the OC release.  Is that your intent @arista-allan ?